src/components/navigation/NavBar.tsx

```diff
@@ -0,0 +1,35 @@
+import { Link, List, ListItem } from "@mui/joy";
+import NextLink from "next/link";
+import { Logo } from "./Logo";
+
+export const NavBar = () => {
+    return (
+        <header>
+            <nav className="flex items-center justify-around flex-wrap bg-blue-500 p-6">
+                <div className="flex items-center flex-shrink-0 text-white mr-6">
+                    <Logo />
+                    <span className="font-semibold text-xl tracking-tight pl-3">
+                        Pocket Assistant
+                    </span>
+                </div>
+
+                <div className="w-full block lg:flex lg:items-center lg:w-auto">
+                    <div className="text-sm lg:flex-grow">
+                        <Link
```

Comment:
```html
                        <Link href="/register">
                            <a className="block mt-4 lg:inline-block lg:mt-0 text-teal-200 hover:text-white mr-4">
                                Register
                            </a>
                        </Link>
```

Reply:
The Component no longer requires manually adding an tag as a child. This behavior was added as an experimental option in version 12.2 and is now the default. In Next.js 13, always renders and allows you to forward props to the underlying tag.